### PR TITLE
Rename "Window" to "Range" where appropriate

### LINKED ISSUE #267: Some methods in viewer are called "window" when they should be called "range"
e.g.:
https://github.com/vais-ral/CILViewer/blob/86b1c569957a30f20f46fb9707b134c1320c6cd1/Wrappers/Python/ccpi/viewer/CILViewer.py#L872

This sets the range of values - i,e, min and max.

The window is a single value =max - min.

Therefore I suggest we rename to setGradientOpacityRange - and similar for other methods in this file.

### PR DESCRIPTION
**Pull request by lauramurgatroyd**
closes #267

### CODE REVIEW

**Comment by lauramurgatroyd** [comment]
I only thought of this afterwards but if you prefer we could use "Limits" instead of "Range" ?

File: Wrappers/Python/ccpi/web_viewer/dev-environment.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # run: conda env create --file environment.yml
-name: cilviewer_webapp
+name: cilviewer_webapp_trame2
```
Comment (Pasarus) [inline Wrappers/Python/ccpi/web_viewer/dev-environment.yml:2]: I don't like this renaming it adds unnecessary length to the name that you'd have to type.

File: Wrappers/Python/ccpi/web_viewer/dev-environment.yml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 dependencies:
 - python==3.9
 - matplotlib # Optional for more colormaps
 - h5py
 - pip
 - pip:
   # Have to install Trame via pip due to unavailability on conda
-  - trame<2.0  # pin version for pip
+  - trame  # pin version for pip
```
Comment (Pasarus) [inline Wrappers/Python/ccpi/web_viewer/dev-environment.yml:10]: Should be handled in the other PR https://github.com/vais-ral/CILViewer/pull/262

**Review by Pasarus — CHANGES_REQUESTED** [review]
A few comments before I look into the code functionality

**Comment by lauramurgatroyd** [comment]
Sorry Sam I didn't mean to push my changes to the environment file! They were from when i was trying to get trame2 working!

File: Wrappers/Python/ccpi/web_viewer/dev-environment.yml (hunk 1 above)
Comment (lauramurgatroyd) [inline Wrappers/Python/ccpi/web_viewer/dev-environment.yml:2]:
```suggestion
name: cilviewer_webapp
```

File: Wrappers/Python/ccpi/web_viewer/dev-environment.yml (hunk 2 above)
Comment (lauramurgatroyd) [inline Wrappers/Python/ccpi/web_viewer/dev-environment.yml:10]:
```suggestion
  - trame<2.0  # pin version for pip
```

**Pull request merged**